jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/provider/ForwardedHeaderFilterSpec.scala

```diff
@@ -0,0 +1,266 @@
+package com.cerner.beadledom.jaxrs.provider
```

Comment:
nit: describe and it blocks needs space after `{`

Reply:
added spaces: https://github.com/cerner/beadledom/pull/149/commits/320a67c5ef71fdcbd15ae8eb0274b250433d5bee